### DISCUSSION TITLE
Body and Surface Forces

### DISCUSSION THREAD

**Discussion by sstevenson638** [ProjectPhysX/FluidX3D#74] [General]
Very cool project. Are the physics just for animation or can they produce reasonably accurate forces and moments on bodies? I am wondering if I could use this as a CFD solver to get body surface loads and forces.

**Comment by ProjectPhysX** [comment]
It's physically accurate. Lattice Boltzmann is a DNS solver an not cheap CGI, yet it's significantly faster than even most CGI solvers. So far I have mainly used (and [thoroughly validated](https://doi.org/10.1186/s43591-021-00018-8)) the software for free surface flows, and I've done a larger variety of other validation cases [here](https://www.researchgate.net/publication/362275548_Accuracy_and_performance_of_the_lattice_Boltzmann_method_with_64-bit_32-bit_and_customized_16-bit_number_formats). Body forces in laminar regime are validated [here](https://doi.org/10.15495/EPub_UBT_00005400), and [preliminary results](https://github.com/ProjectPhysX/FluidX3D/discussions/58#discussioncomment-5192266) show that body forces are accurate in the highly turbulent regime as well.

**Reply by sstevenson638** [reply]
Thanks for the reply! I have a lot of exploring and learning to do.
